### DISCUSSION TITLE
How can I change the drop indicator of pragmatic-drag-and-drop-react-beautiful-dnd-migration

### DISCUSSION THREAD

**Discussion by one-pieces** [atlassian/pragmatic-drag-and-drop#42] [Q&A] [answered]
Hi, thanks for supporting migration, it's very great.

There is a question from me, that I want to change the appearance of drop indicator but I could not find out the solution.

Could you tell how I can do that?

**Comment by alexreardon** [comment, marked as answer]
Unfortunately the appearance of the drop indicator in our migration package is locked down. Right now the purpose of our migration package is to be a temporary step towards Pragmatic drag and drop. If you want full control over the UI, including drop indicators, then you will need to use our standard packages

**Comment by pramodrwt** [comment]
you can change color with css

:root {
  --ds-border-brand: #000 !important;
}
